tools/gulp/packaging/build-utils.ts

```diff
@@ -0,0 +1,85 @@
+import {writeFileSync, readFileSync} from 'fs-extra';
+import {join} from 'path';
+import {LICENSE_BANNER, MATERIAL_VERSION} from '../constants';
```

Comment:
Not sure I'm a bit afraid of having 5 files just for small functions. What do you think?